src/aat/resources/features/F-1008 - Submit Event Creation Handle Case Links/CreateCases/Caseworker/F-1008-GetUpdateEventToken.td.json

```diff
@@ -0,0 +1,46 @@
+{
+	"title": "should create an event token for correct inputs",
+	"_guid_": "F-1008-GetUpdateEventToken",
+
+	"productName": "CCD Data Store",
+	"operationName": "Retrieve update token for an existing case",
+
+	"method": "GET",
+	"uri": "/caseworkers/{uid}/jurisdictions/{jid}/case-types/{ctid}/cases/{cid}/event-triggers/{etid}/token",
```

Comment:
If a **common TD file** to cover the basics of the API signature does not exist can we create on and use it as this endpoint is being used multiple times in these tests:

* F-1008-GetLinkedCaseUpdateEventToken
* F-1008-GetMultipleLinkedCaseUpdateEventToken
* F-1008-GetUpdateEventToken